release_notes/release_notes.html

```diff
@@ -1,5 +1,10 @@
-<b>Microsoft OneDrive Release Notes - Published by Splunk January 27, 2022</b>
+<b>Microsoft OneDrive Release Notes - Published by Splunk March 15, 2022</b>
```

Comment:
minor: should we update the date here to be today? (i.e. we are releasing this app today, right?)

reference: https://github.com/splunk-soar-connectors/panorama/commit/25641b432e585ab247877277644d405513e8af01

Reply:
Nice catch!